accessibility/accessibility-in-emails.md

```diff
@@ -0,0 +1,42 @@
+# Accessibility in emails
+
+This info sits alongside our general guide to writing emails at ????.
+
+## Layout
+
+- Rich visual emails typically use tables for layout.
+- ??? Setting `role="presentation"` seems like a good idea, but some ppl recommend against it. 
```

Comment:
people really recommend against it?! wow. why?

Reply:
Oh wait no, this is me getting confused with another ARIA suggestion. It looks good.